cluster/manifests/skipper/hpa.yaml

```diff
@@ -21,3 +21,14 @@ spec:
     resource:
       name: memory
       targetAverageUtilization: {{ .ConfigItems.skipper_ingress_target_average_utilization_memory }}
+  behavior:
+    scaleDown:
+      stabilizationWindowSeconds: 600
```

Comment:
Is the default of 5 minutes not good enough?

Reply:
It's the default and better slower in this case for workloads that are spiking forth and back